.github/dependabot.yml

```diff
@@ -52,4 +52,4 @@ updates:
   allow:
   - dependency-type: "direct"
   versioning-strategy: "increase"
-  target-branch: v14-branch
+  target-branch: v15-branch
```

Comment:
No longer need 15 it seems: https://docs.altis-dxp.com/releases/

Reply:
True. I'll create another set of PRs to remove v15!